includes/functions/admin/conditionals.php

```diff
@@ -104,7 +104,7 @@ function pum_is_submenu_page( $key = null ) {
 		pum_is_admin_page(),
 		! pum_is_popup_editor(),
 		! pum_is_popup_theme_editor(),
-		$key && ! empty( $GLOBALS['hook_suffix'] ) ? $GLOBALS['hook_suffix'] === PUM_Admin_Pages::get_page( $key ) : true,
+		$key && ! empty( $GLOBALS['hook_suffix'] ) ? $GLOBALS['hook_suffix'] === PUM_Admin_Pages::get_page( $key ) : true,  //  phpcs:ignore
```

Comment:
@s-acv2 - What do you think it should be?

Quick tip:

`PUM_Admin_Pages::get_page` is calling a function in the `PUM_Admin_Pages` class. The return value is what is being compared. Just flip the $GLOBALS and PUM_Admin_Pages function calls.

```
$key && ! empty( $GLOBALS['hook_suffix'] ) ? PUM_Admin_Pages::get_page( $key ) === $GLOBALS['hook_suffix'] : true,